static/mobile.css

```diff
@@ -0,0 +1,56 @@
+@media only screen and (max-device-width: 750px) {
```

Comment:
Why 750px?

This seems to work okay on mobiles in both orientation, but on some tablets (Nook, Fire, iPad/iPad mini), it looks weird in portrait mode because the tablet is 768 screen pixels.  Namely, the hero columns are close together (they need the margin-bottom at line 14).